backend/src/packages/trucks/truck.controller.ts

```diff
@@ -30,17 +30,17 @@ import { type TruckService } from './truck.service.js';
  *         manufacturer:
  *           type: string
  *           enum:
- *             - miller_industries
- *             - mercedes_benz
- *             - daf
- *             - man
- *             - volvo
- *             - renault
- *             - iveco
- *             - scania
- *             - kenworth
- *             - ford
- *           example: kenworth
+ *             - Miller_industries
```

Comment:
What is the purpose of changing the case? The frontend will have to handle it anyway to remove the _, and we can change the case of the first letter via css.